osf/utils/migrations.py

```diff
@@ -11,17 +13,33 @@
 logger = logging.getLogger(__file__)
 
 
+def get_osf_models():
+    """
+    Helper function to retrieve all osf related models.
+
+    Example usage:
+        with disable_auto_now_fields(models=get_osf_models()):
+            ...
+    """
+    return list(itertools.chain(*[app.get_models() for app in apps.get_app_configs() if 'addons' in app.label or 'osf' in app.label]))
```

Comment:
The check could be more precise, e.g, `app.label.startswith('addons_')`, so we don't get bit by something like `django_model_addons`.